projects/plugins/jetpack/changelog/add-enable-userless

```diff
@@ -0,0 +1,4 @@
+Significance: major
+Type: enhancement
+
+Activates Jetpack without requiring a authorized WordPress.com user
```

Comment:
```suggestion
Activates Jetpack without requiring connecting a WordPress.com user account
```

Just a suggestion as it's more consistent with our corresponding prompts/buttons etc that ask a user to connect their WordPress.com account